test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -61,4 +59,113 @@ public void FirstExperienceWithML()
             });
         }
     }
+
+    public class GithubClassification
+    {
+        public void ClassifyGithubIssues()
+        {
+            var env = new TlcEnvironment(new SysRandom(0), verbose: true);
+
+            string dataPath = "corefx-issues-train.tsv";
+
+            // Create reader with specific schema. 
+            // string :ID, string: Area, string:Title, string:Description
+            var reader = TextLoader.CreateReader(env, ctx =>
+                                          (area: ctx.LoadText(1),
+                                            title: ctx.LoadText(2),
+                                            description: ctx.LoadText(3),
+                                            dataSource,
+                                            useHeader: true));
+
+            var data = reader.Read(dataPath);
+
+            var estimator = data.MakeEstimator()
```

Comment:
Do I call `MakeEstimator()` on `data` or `reader`? These examples seen to do it on both.

Reply:
You can actually do it on both, but I think we should do it consistently in the samples

---
In reply to: [215321435](https://github.com/dotnet/machinelearning/pull/812#discussion_r215321435) [](ancestors = 215321435)